iroh-rpc/src/builder.rs

```diff
@@ -67,4 +71,20 @@ impl<T> RpcConfig<T> {
         self.client.addrs.insert(name.into(), (addr, peer_id));
         self
     }
+
+    pub fn build(self) -> Result<(Client, Server<T>), RpcError> {
+        let (sender, receiver) = mpsc::channel(0);
```

Comment:
are you sure this should be 0?

Reply:
no no idea, that was a placeholder. Not sure what good conventions around this are. Would love some advice.

Reply:
I would suggest something like 64 to start with, and potentially making it configurable later

Reply:
oh I had a thought. When I was playing around with the # of channels that work for the stream with the mem transport without failure, I got to about 1000, which is where I left it. Wouldn't that imply that this too would have to be somewhere in the same realm of about 1000 for it to handle streaming?  

definitely will have a todo about making it configurable.

Reply:
I still have a bunch of work to do around streaming to trigger a slow down when the streaming channel gets filled, rather than error & stop like it does now.